config/database.js

```diff
@@ -1,12 +1,22 @@
 const mongoose = require('mongoose')
 
-mongoose.connect('mongodb://127.0.0.1/WeAreYourTeam')
+const dbURI = 'mongodb://127.0.0.1/WeAreYourTeam' //should put it to .env
+
+mongoose.connect(dbURI)
 
 mongoose.Promise = Promise
 
 const db = mongoose.connection
 
-// Bind connection to error event (to get notification of connection errors)
+db.on('connected', () => {
+  console.log('MongoDB connection connected to ' + dbURI)
+})
+
+db.on('disconnected', () => {
+  console.log('MongoDB connection disconnected')
+})
 
-// eslint-disable-next-line no-console
-db.on('error', console.error.bind(console, 'MongoDB connection error:'))
+db.on('error', (err) => {
+  console.log('MongoDB connection error. Please try to start/restart your MongoDB')
```

Comment:
Could you change this to `console.error`? Let's direct error messages to proper channels